src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -269,6 +269,18 @@ The term "lock file" comes from NuGet. For example, search for "UnauthorizedLock
     <value>Using the shared runtime and Android App Bundles at the same time is not currently supported. Use the shared runtime for Debug configurations and Android App Bundles for Release configurations.</value>
     <comment>The following are literal names and should not be translated: Android App Bundles</comment>
   </data>
+  <data name="XA0119_AOT" xml:space="preserve">
+    <value>Using Fast Deployment and AOT at the same time is not recommended. Use Fast Deployment for Debug configurations and AOT for Release configurations.</value>
+    <comment>The following are literal names and should not be translated: Fast Deployment, AOT, Debug, Release.</comment>
+  </data>
+  <data name="XA0119_LinkMode" xml:space="preserve">
+    <value>Using Fast Deployment and the Linker at the same time is not recommended. Use Fast Deployment for Debug configurations and the Linker for Release configurations.</value>
+    <comment>The following are literal names and should not be translated: Fast Deployment, Linker, Debug, Release.</comment>
+  </data>
+  <data name="XA0119_LinkTool" xml:space="preserve">
+    <value>Using Fast Deployment and a Code Shrinker at the same time is not recommended. Use Fast Deployment for Debug configurations and a Code Shrinker for Release configurations.</value>
+    <comment>The following are literal names and should not be translated: Fast Deployment, Code Shrinker, Debug, Release.</comment>
+  </data>
```

Comment:
I don't actually know if all of these should be "literal names":
* Fast Deployment
* Linker
* Code Shrinker

I'm pretty sure AOT should be a "literal name", though.

Reply:
Agreed on preserving "AOT" as a literal name because it's an abbreviation that has broader uses in the industry outside of Xamarin. For the other terms, my recommendation at this time is to lowercase all of them ("fast deployment", "linker", "code shrinker") and not list them as literal names in the _.resx_ file comments.

This aligns with some existing messages in the _.resx_ file, like XA3001 for AOT, XA1011 for code shrinker, and XA0119_AAB for shared runtime.

### Background info

Corresponding IDE translations of these terms are not consistent across languages or IDEs. Using lowercase non-literal names therefore seems like the best compromise for now to help users discover the corresponding settings in the IDE. On top of that, as hinted at by the differences across the IDEs in the original en-US strings, none of these three features have names selected by a marketing team. (Names selected by a marketing team would more often be important to preserve as literal names.)

| loc | Windows | macOS | 
| --- | ------- | ----- | 
| en-US | Code shrinker | Code Shrinker | 
| cs-CZ | Nástroj pro zmenšování kódu | Nástroj pro zmenšení kódu |
| de-DE | Codeshrinker | Codeshrinker |
| es-ES | Reductor de código | Reductor de código |
| fr-FR | Réducteur de code | Réducteur de code |
| it-IT | Code shrinker | Code shrinker |
| ja-JP | コード shrinker | コード shrinker |
| ko-KR | 코드 Shrinker | 코드 Shrinker | 
| pl-PL | Narzędzie do zmniejszania objętości kodu | Narzędzie do zmniejszania kod | 
| pt-BR | Redutor de código | Redutor de Código |
| ru-RU | Средство сжатия кода | Средство сжатия кода |
| tr-TR | Kod sıkıştırıcısı | Kod Sıkıştırıcı |
| zh-CN | 代码 shrinker | 代码 Shrinker |
| zh-TW | 程式碼壓縮工具 | 程式碼 Shrinker |

| loc | Windows | macOS | 
| --- | ------- | ----- | 
| en-US | Use Fast Deployment (debug mode only) | Fast Assembly Deployment|
| cs-CZ | Používat Fast Deployment (pouze režim ladění) | Rychlé nasazení sestavení |
| de-DE | Fast Deployment verwenden (nur Debugmodus) | Schnelle Assemblybereitstellung |
| es-ES | Usar Fast Deployment (solo modo de depuración) | Implementación rápida de ensamblados |
| fr-FR | Utiliser Fast Deployment (mode débogage uniquement) | Déploiement d’assembly rapide |
| it-IT | Usa Fast Deployment (solo modalità di debug) | Distribuzione rapida di assembly |
| ja-JP | Fast Deployment の使用 (デバッグ モードのみ) | 迅速なアセンブリの配置 |
| ko-KR | 빠른 배포 사용(디버그 모드 전용) | 빠른 어셈블리 배포 |
| pl-PL | Użyj opcji Fast Deployment (tylko w trybie debugowania) | Szybkie wdrażanie zestawu |
| pt-BR | Usar Fast Deployment (somente modo de depuração) | Implantação de Assembly Rápida |
| ru-RU | Использовать Fast Deployment (только в режиме отладки) | Быстрое развертывание сборок |
| tr-TR | Fast Deployment'ı kullan (yalnızca hata ayıklama modu) | Hızlı Bütünleştirilmiş Kod Dağıtımı |
| zh-CN | 使用 Fast Deployment (仅调试模式) | 快速程序集部署 |
| zh-TW | 使用 Fast Deployment (僅限偵錯模式) | 快速部署組件 |

| loc | Windows | macOS | 
| --- | ------- | ----- | 
| en-US | Linking | Linker Behaviour |
| cs-CZ | Propojení | Chování linkeru |
| de-DE | Verknüpfen | Linkerverhalten |
| es-ES | Vinculando | Comportamiento del enlazador |
| fr-FR | Édition des liens en cours | Comportement de l’éditeur de liens (Linker) : |
| it-IT | Collegamento | Comportamento del linker |
| ja-JP | リンク中 | リンカーの動作 |
| ko-KR | 연결하는 중 | 링커 동작 |
| pl-PL | Łączenie | Zachowanie konsolidatora |
| pt-BR | Vinculação | Comportamento do Vinculador |
| ru-RU | Связывание | Поведение компоновщика |
| tr-TR | Bağlama | Bağlayıcı Davranışı |
| zh-CN | 链接 | 链接器行为 |
| zh-TW | 正在連結 | 連結器行為 |